uaaextras/templates/email/body-expiring-password.html

```diff
@@ -11,7 +11,8 @@
             will expire in {{password.daysUntilExpiration}} days.
         </p>
         <p>
-            Please <a href="{{password.loginLink}}">log in</a> with your current password</a> and then <a href="{{password.changeLink}}">change your password</a>.
+            First, <a href="{{password.loginLink}}">log in</a> with your current password.
+            After you login, use this <a href="{{password.changeLink}}">link</a> to change your password.
```

Comment:
I took a shot at updating the link text to make it reflect the content of the link.